std_msgs/MultiArrayLayout  layout
float32[] x
float32[] y
float32[] z

---

std_msgs/MultiArrayLayout layout
float32[] x
float32[] y
float32[] z